exercises/circular-buffer/circular_buffer_test.py

```diff
@@ -7,40 +7,37 @@
 )
 
 
+# Tests adapted from `problem-specifications//canonical-data.json` @ v1.0.1
 class CircularBufferTest(unittest.TestCase):
     def test_read_empty_buffer(self):
         buf = CircularBuffer(1)
         with self.assertRaises(BufferEmptyException):
             buf.read()
 
+    def test_read_just_written_item(self):
+        buf = CircularBuffer(1)
+        buf.write('1')
+        self.assertEqual('1', buf.read())
+
     def test_write_and_read_back_one_item(self):
         buf = CircularBuffer(1)
         buf.write('1')
         self.assertEqual('1', buf.read())
         with self.assertRaises(BufferEmptyException):
             buf.read()
 
-    def test_write_and_read_back_multiple_items(self):
+    def test_write_and_read_back_multiple_items_ordered(self):
         buf = CircularBuffer(2)
         buf.write('1')
         buf.write('2')
-        self.assertEqual(buf.read(), '1')
-        self.assertEqual(buf.read(), '2')
-        with self.assertRaises(BufferEmptyException):
-            buf.read()
+        self.assertEqual('1', buf.read())
+        self.assertEqual('2', buf.read())
 
-    def test_clearing_buffer(self):
-        buf = CircularBuffer(3)
-        for c in '123':
-            buf.write(c)
-        buf.clear()
-        with self.assertRaises(BufferEmptyException):
-            buf.read()
+    def test_full_buffer_cant_written(self):
+        buf = CircularBuffer(1)
         buf.write('1')
-        buf.write('2')
-        self.assertEqual(buf.read(), '1')
-        buf.write('3')
-        self.assertEqual(buf.read(), '2')
+        with self.assertRaises(BufferFullException):
+            buf.write('2')
 
     def test_alternate_write_and_read(self):
         buf = CircularBuffer(2)
```

Comment:
Looks like this test has been changed in the canonical data - it should be a `CircularBuffer(1)`.

Reply:
Changed that. Thanks